data/model/test/test_repo_mirroring.py

```diff
@@ -255,3 +255,21 @@ def test_repo_mirror_robot(initialized_db):
     assert mirror
     assert mirror.internal_robot
     assert model.repo_mirror.robot_has_mirror(mirror.internal_robot)
+
+
+def test_release_mirror_success_status(initialized_db):
+    disable_existing_mirrors()
+    mirror, repo = create_mirror_repo_robot(["updated", "created"], repo_name="first")
+    mirror = release_mirror(mirror, sync_status=RepoMirrorStatus.SUCCESS)
+    assert mirror.sync_retries_remaining == 3
+
+    mirror = release_mirror(mirror, sync_status=RepoMirrorStatus.FAIL)
+    assert mirror.sync_retries_remaining == 2
+
+    mirror = release_mirror(mirror, sync_status=RepoMirrorStatus.FAIL)
+    assert mirror.sync_retries_remaining == 1
+
+    mirror = release_mirror(mirror, sync_status=RepoMirrorStatus.FAIL)
+    original_sync_start_date = mirror.sync_start_date
+    assert mirror.sync_retries_remaining == 3
+    assert mirror.sync_start_date == original_sync_start_date
```

Comment:
Not sure what this is testing? You assigned `original_sync_start_date = mirror.sync_start_date` a few lines above, so we would expect this to always hold anyways.

Reply:
Yes my mistake, I meant to check `assert mirror.sync_retries_remaining == 3` for it before calling the function. I've pushed the changes.